apps/crn-server/src/data-providers/research-outputs.data-provider.ts

```diff
@@ -162,6 +163,7 @@ export class ResearchOutputSquidexDataProvider
       environments: environmentIds,
       organisms: organismIds,
       subtype: (subtypeId && [subtypeId]) || [],
+      addedDate: addedDate || new Date(Date.now()).toISOString(),
```

Comment:
is this the same functionality that we added to the controller? https://github.com/yldio/asap-hub/blob/e1286c235c1d078a82e434bc1756a2b9ff31fe3d/apps/crn-server/src/controllers/research-outputs.ts#L91?

This defaulting behaviour should imho only sit in the controller. The data-provider input type should therefore require that the `addedDate` is obligatory.

Reply:
Yes I was asking myself where would be the best place for it. I ended up putting it at both provider and controller considering they could be called by different places that wouldn't pass the date. Will apply it for the controller only.